### PR TITLE
Nowcasting concept tweaks

### PR DESCRIPTION
**Pull request by seabbs**

### CODE REVIEW

File: sessions/nowcasting.qmd [hunk 1]
```diff
@@ -159,11 +159,17 @@ ggplot(combined_cutoffs, aes(x = day, y = onsets, colour = cutoff)) +
 As we can see, even though it may much seem like the epidemic curve is going down, in fact in the final data set one can see that at the time symptom onsets were still increasing.
 The apparent decline towards the present (indicated by a dashed vertical line) was caused by the delay in reporting.
 
-Why then, you might ask, not just plot the data by date of reporting which correctly showed the data to be still increasing and should, by definition, not be subject to future changes?
+::: {.callout-note}
+## Why then, you might ask, not just plot the data by date of reporting which correctly showed the data to be still increasing and should, by definition, not be subject to future changes?
+
 This can sometimes be a sensible way to visualise the data.
-However, reporting might itself be subject to biases such as breaks during the weekend, holidays etc.
+However, reporting might itself be subject to biases such as breaks during the weekend, holidays etc or reporting delays may vary over timel
```
Comment (kathsherratt) [inline sessions/nowcasting.qmd:166]:
```suggestion
However, reporting might itself be subject to biases such as breaks during the weekend, holidays etc or reporting delays may vary over time
```

**Review by kathsherratt — APPROVED** [review]

File: sessions/nowcasting.qmd (hunk 1 above)
Comment (sbfnk) [reply sessions/nowcasting.qmd:166]: and a full stop

File: sessions/nowcasting.qmd [hunk 2]
```diff
@@ -159,11 +159,17 @@ ggplot(combined_cutoffs, aes(x = day, y = onsets, colour = cutoff)) +
 As we can see, even though it may much seem like the epidemic curve is going down, in fact in the final data set one can see that at the time symptom onsets were still increasing.
 The apparent decline towards the present (indicated by a dashed vertical line) was caused by the delay in reporting.
 
-Why then, you might ask, not just plot the data by date of reporting which correctly showed the data to be still increasing and should, by definition, not be subject to future changes?
+::: {.callout-note}
+## Why then, you might ask, not just plot the data by date of reporting which correctly showed the data to be still increasing and should, by definition, not be subject to future changes?
```
Comment (sbfnk) [inline sessions/nowcasting.qmd:163]:
```suggestion
## Plotting and analysing data by report date
Looking at the plots one might suggest plotting and analysing the data by date of reporting which correctly showed the data to be still increasing and which should, by definition, not be subject to future changes.
```
Just worried the long sentence will render weirdly.

File: sessions/nowcasting.qmd [hunk 3]
```diff
@@ -178,7 +184,9 @@ proportion_reported <- plnorm(1:15, 1, 0.5)
 plot(proportion_reported)
 ```
 
+::: {.callout-tip}
```
Comment (sbfnk) [inline sessions/nowcasting.qmd:187]:
```suggestion
::: {.callout-tip}
## The `plnorm` function
```

**Review by sbfnk — APPROVED** [review]

**Pull request merged**